scripts/wonderwall.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 
-./.wonderwall/wonderwall-master/bin/wonderwall \
+./.wonderwall/wonderwall \
```

Comment:
Var ikke klar for den spennende bruken av wonderwall i pam-stillingsok. Gleder meg til å få den forklart på torsdag :)